hil-test/tests/delay.rs

```diff
@@ -37,25 +36,33 @@ mod tests {
     }
 
     #[test]
-    #[timeout(1)]
+    #[timeout(2)]
     fn delay_ns(mut ctx: Context) {
         let t1 = esp_hal::time::current_time();
         ctx.delay.delay_ns(600_000_000);
         let t2 = esp_hal::time::current_time();
 
         assert!(t2 > t1);
-        assert!((t2 - t1).to_nanos() >= 600_000_000u64);
+        assert!(
+            (t2 - t1).to_nanos() >= 600_000_000u64,
+            "diff: {:?}",
+            (t2 - t1).to_nanos()
+        );
     }
 
     #[test]
-    #[timeout(1)]
+    #[timeout(2)]
```

Comment:
There is some weirdness here. This test finishes in ~1.23s. Considering a simple `assert!(1+1=2)` takes ~0.6s, I hope the timeout I saw here isn't caused by a still-wrong tick rate.